content/blog/2022-01-31-january-22-community-gems.md

```diff
@@ -0,0 +1,179 @@
+
+Thanks for the question @alphaomega!
+
+The best way to handle any package dependencies is to include a requirements.txt
+file with the specific versions your pipeline needs.
+
+Another approach you can take is having a stage that dumps the package version
+as an intermediate output. It doesn't have to be saved in Git or DVC because
+it's easily reproduced and the DVC internals should be able to take care of
+detecting that the package didn't change.
+
+### [Does DVC track dependencies which are in the `dvc.yaml` pipeline?](https://discord.com/channels/485586884165107732/563406153334128681/920659549835370497)
+
+Thanks for another great question @rie!
+
+DVC doesn't track the pipeline dependencies, only the outputs. If you want DVC
+to track a pure data dependency that's not an output of a different stage, you
+need to track it with `dvc add ...`
+
+### [What is the difference between Kubeflow pipelines and DVC pipelines?](https://discord.com/channels/485586884165107732/563406153334128681/922728960478035978)
+
+This is a fantastic question! Thanks for asking @ramakrishnamamidi!
+
+A major difference is that DVC focuses primarily on ML _development_ and adding
+lightweight functionality on top of existing projects, which may be reusable in
+deployment in some cases.
+
+Kubeflow focuses on _deployment_ and building on top of Kubernetes, which could
+be used during development but requires more up-front effort.
+
+### [Could DVC be a good alternative to LFS for game development?](https://discord.com/channels/485586884165107732/485586884165107734/928336349487067196)
+
+Thanks for such an interesting question @CB!
+
+Yes! We have community members that use DVC to handle their large files in game
+development.
+
+There are several other use cases we've seen for DVC outside of machine learning
+and data science. Some people have used DVC to track build artifacts for
+deployment systems and to track performance data alongside design iterations and
+simulation tools.
+
+You should check out our
+[#beyond-ml](https://discord.com/channels/485586884165107732/918159153824952320)
+Discord channel to stay up to date with the other use cases the community is
+coming p with!
+
+### [Does DVC run on JSON/YAML configuration files for all things?](https://discord.com/channels/485586884165107732/563406153334128681/928779586622332938)
+
+This is a great question about large projects with a lot of dependencies from
+@SolemnSimulacrum!
+
+All of the dependencies you list in `dvc run` are in fact configured in the
+`dvc.yaml` file. `dvc run` is a convenience for adding a pipeline stage to this
+file and then doing `dvc repro` on that stage. It's completely acceptable and
+even encouraged to directly edit `dvc.yaml` if that's easier.
+
+For example, if you are currently executing a command like this:
+
+```dvc
+dvc run -n prune \
```

Comment:
Could you add an `out` and `cmd` to this? `cmd` is required and at least one `out` is almost always used, so it would help make a slightly more realistic example.

Reply:
Done!